src/pybind/mgr/dashboard/frontend/src/app/shared/components/helper/helper.component.html

```diff
@@ -5,7 +5,7 @@
   <ng-content></ng-content>
 </ng-template>
 
-<cds-tooltip [description]="popoverTpl">
+<cds-tooltip [description]="popoverTpl" [autoAlign]="true">
```

Comment:
to fix the lint issues.
```suggestion
<cds-tooltip [description]="popoverTpl"
             [autoAlign]="true">
```

Reply:
changes are done